docs/zero-to-hero/01-basics/03-hashing-and-unit-tests.md

```diff
@@ -25,13 +25,38 @@ As mentioned in the first section of this **Basics** chapter, our smart contract
 We'll add a dependency to the [hex crate](https://crates.io/crates/hex) to make things easier. As you may remember, dependencies live in the manifest file.
 
 ```rust reference
-https://github.com/mikedotexe/crossword-snippets/blob/2069dcfd5419570ba4b54321005f674bb6f84224/Cargo.toml#L10-L12
+https://github.com/near-examples/crossword-tutorial-chapter-1/blob/94f42e75cf70ed2aafb9c29a1faa1e21f079a49e/contract/Cargo.toml#L10-L12
 ```
 
-Let's write a unit test that acts as a helper during development. This unit test will sha256 hash the input **"near nomicon ref finance"** and print it in a human-readable, hex format.
-
-```rust reference
-https://github.com/mikedotexe/crossword-snippets/blob/2069dcfd5419570ba4b54321005f674bb6f84224/src/lib.rs#L45-L62
+Let's write a unit test that acts as a helper during development. This unit test will sha256 hash the input **"near nomicon ref finance"** and print it in a human-readable, hex format. (We'll typically put unit tests at the bottom of the `lib.rs` file.)
+
+```rust
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use near_sdk::test_utils::{get_logs, VMContextBuilder};
+    use near_sdk::{testing_env, AccountId};
+
+    // part of writing unit tests is setting up a mock context
+    // provide a `predecessor` here, it'll modify the default context
+    fn get_context(predecessor: AccountId) -> VMContextBuilder {
+        let mut builder = VMContextBuilder::new();
+        builder.predecessor_account_id(predecessor);
+        builder
+    }
+
+    #[test]
+    fn debug_get_hash() {
+        // Basic set up for a unit test
+        testing_env!(VMContextBuilder::new().build());
```

Comment:
Confused why get_context is required if not used?

Reply:
Good question. I saw a contributor ran into confusion, but perhaps I'll remove that function from this section and include it in the second unit test that actually uses it. Thanks

Reply:
Thanks again, Austin. I've pushed the update here and am much happier with it:
https://github.com/near/sdk-docs/pull/71/commits/56a9fd9afdf59187ad1b581d87e44b3cb9093889